README.md

```diff
@@ -82,6 +82,9 @@ with tables based on `pangeo_forge_orchestrator.models`. If a file already exist
 to refect the tables defined by `pangeo_forge_orchestrator.models`. Note that `.sqlite` files are excluded by `.gitignore`, so you don't
 need to worry about the database file appearing in your commit.
 
+**Note:** Changes to modules in `pangeo_forge_orchestrator/*` may require that the package be re-installed with `pip install '.[dev]'`
+before running local tests.
+
```

Comment:
Maybe this is just a basic pytest assumption, in which case we can remove. But I kept forgetting to do this and spent unnecessary amounts of time wondering why the tests were failing. 🤦

Reply:
If you run `pip install -e '.[dev]'` once, this should be unnecessary.